packages/web-components/@sfx/autocomplete/src/autocomplete.ts

```diff
@@ -77,14 +138,16 @@ export default class Autocomplete extends LitElement {
   }
 
   /**
-   * Dispatches an [[AUTOCOMPLETE_ACTIVE_TERM]] event with the Sayt autocomplete term.
-   *
-   * @param event An event that contains a Sayt autocomplete term.
+   * Dispatches an [[AUTOCOMPLETE_ACTIVE_TERM]] event with the selected term.
    */
-  handleHoverTerm(event: MouseEvent): void {
-    const target = event.target as HTMLElement;
-    if (target.tagName.toLowerCase() !== 'li') return;
-    const term = target.innerText;
+  dispatchSelectedTerm(): void {
+    if (this.selectedIndex < 0 || this.selectedIndex >= this.length) return;
```

Comment:
I would have liked to have written this as `this.selectedIndex < 0 || this.length <= this.selectedIndex` to better emphasize the range, but our `yoda` settings currently don't allow ranges. I may submit an amendment to the linter settings in the near future.